packages/contracts-bedrock/scripts/merkle-trie-fuzzer/README.md

```diff
@@ -0,0 +1,22 @@
+# `merkle-trie-fuzzer`
+
+A lightweight, efficient Merkle Trie fuzzer for use in Optimism bedrock's [Merkle Trie Verifier](https://google.com).
```

Comment:
Looks like a wrong link?

Reply:
Oops- yep, forgot to fill the placeholder. Updated 🤝